app/styles/ui/dialogs/_rebase-conflicts.scss

```diff
@@ -0,0 +1,124 @@
+@import '../../mixins';
+
+dialog#rebase-conflicts-list {
+  width: 500px;
+
+  .summary {
+    margin-bottom: 20px;
+  }
+
+  .dialog-header h1 {
+    font-weight: var(--font-weight-light);
+  }
+
+  .dialog-content {
+    padding-bottom: var(--spacing);
+  }
+
+  .green-circle {
+    background-color: var(--color-new);
+    color: var(--background-color);
+    border-radius: 50%;
+    height: 22px;
+    width: 22px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0;
+    max-height: 285px;
+    overflow-y: auto;
+
+    li {
+      margin-bottom: var(--spacing-double);
+      padding-left: 0;
+    }
+
+    li:last-of-type {
+      margin-bottom: calc(var(--spacing) * 2);
+    }
+
+    li.unmerged-file-status-resolved,
+    li.unmerged-file-status-conflicts {
+      display: flex;
+      flex-flow: row nowrap;
+
+      .file-octicon {
+        margin-right: var(--spacing);
+      }
+
+      .column-left {
+        display: flex;
+        flex-flow: column nowrap;
+        align-items: start;
+        max-width: 50%;
+        padding-right: var(--spacing);
+
+        .path-text-component {
+          text-overflow: ellipsis;
+          max-width: 100%;
+        }
+      }
+
+      .action-buttons {
+        margin-left: auto;
+        flex-shrink: 0;
+        flex: 0 1 auto;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .green-circle:last-child {
+        margin-left: auto;
+        margin-top: var(--spacing);
+        flex-shrink: 0;
+        flex: 0 1 auto;
+      }
+    }
+
+    .unmerged-file-status-resolved .file-conflicts-status {
+      color: $green;
```

Comment:
Are the use of color constants being tracked in some other issue? We should never be using primer SCSS variables outside of `_variables.scss`. I suggest we change these right now to use our variables such that it's themeable.

```suggestion
      color: var(--color-new);
```

Side-note, I thought we had to do `#{$green}` these days?

Reply:
> We should never be using primer SCSS variables outside of `_variables.scss` 

Agreed. I crimped these styles from `app/styles/ui/dialogs/_merge-conflicts.scss` and these styles will go away in #7030 after merging them together, but I'll clean this up and check for any other usages that we should tidy up outside of this PR.

> Side-note, I thought we had to do #{$green} these days?

Me too, and I'm surprised this didn't get raise any issues. As a sanity check I looked at the CSS from `1.6.3` and these are transformed fine:

<img width="999" src="https://user-images.githubusercontent.com/359239/54037265-17b1bc00-4194-11e9-94c6-34ce833f84ff.png">